examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2466,6 +2471,122 @@ impl FrontendNode {
         expect_matches!(result, Err(_));
         Ok(())
     }
+
+    fn test_pseudo_node_privilege(&mut self) -> TestResult {
+        // gRPC server pseudo node can be created even with Top confidentiality.
+        {
+            let top_label = oak_abi::label::confidentiality_label(oak_abi::label::top());
+            let config = oak::node_config::grpc_server(ADDITIONAL_TEST_SERVER_ADDR);
+            let (wh, rh) =
+                oak::channel_create("Test", &top_label).expect("could not create channel");
+            expect_eq!(
+                Ok(()),
+                oak::node_create("grpc_server", &config, &top_label, rh)
+            );
+            expect_eq!(Ok(()), oak::channel_close(rh.handle));
+            expect_eq!(Ok(()), oak::channel_close(wh.handle));
+        }
+
+        // HTTP server pseudo node can be created even with Top confidentiality.
```

Comment:
I guess this is because we don't currently have a more precise privilege to assign it, so it has infinite privilege, right? Could you clarify in the comment? Also link to some issue about reducing its privilege, if we have it.

Reply:
Added TODOs that reference the issue.